src/pages/VenuePage/TemplateWrapper.tsx

```diff
@@ -57,7 +79,11 @@ const TemplateWrapper: React.FC<TemplateWrapperProps> = ({ venue }) => {
 
     case VenueTemplate.partymap:
     case VenueTemplate.themecamp:
-      template = <PartyMap venue={venue} />;
+      template = (
+        <Suspense fallback={<></>}>
+          <PartyMap venue={venue} />
+        </Suspense>
+      );
```

Comment:
As per previous comment, this should probably be reverted due to conflicts with:

- https://github.com/sparkletown/sparkle/pull/1737

Reply:
Done in b7ec26dc13ee3c39f4d061fe3565ff7cd02b0f3e